### PR TITLE
Announcement adjustments

### LINKED ISSUE #20: inconsistante with using "treasure" and "flag" in announcements.

### PR DESCRIPTION
**Pull request by Uni-tato**
Just some very minor things.
fixes #20

### CODE REVIEW

File: src/com/hiuni/milkwars/events/ClanKillCounterEvent.java [hunk 1]
```diff
@@ -51,11 +51,11 @@ public static void onPlayerDeath (PlayerDeathEvent event) {
             return;
         }
 
-        event.setDeathMessage(Announce.formatPlayerName(player, friendlyClan) + " of the " +
-                friendlyClan.getName() + " clan died at the hands of " +
-                Announce.formatPlayerName(killer, enemyClan) + " and the " +
-                enemyClan.getName() + " clan.");
-        enemyClan.addKill();
```
Comment (histefanhere) [inline src/com/hiuni/milkwars/events/ClanKillCounterEvent.java:58]: You commented out the addkill method lol

**Review by histefanhere — CHANGES_REQUESTED** [review]
wrong

File: src/com/hiuni/milkwars/events/ClanKillCounterEvent.java (hunk 1 above)
Comment (Uni-tato) [reply src/com/hiuni/milkwars/events/ClanKillCounterEvent.java:58]: whoops

**Review by histefanhere — APPROVED** [review]
Looks good now

**Pull request merged**